# Message for online repair result
bool repair_needed
string slugsin_location